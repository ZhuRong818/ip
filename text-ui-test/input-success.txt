todo read
deadline submit /by Friday
event meeting /from Monday 2pm /to 4pm
list
mark 2
unmark 2
mark 1
list
bye
