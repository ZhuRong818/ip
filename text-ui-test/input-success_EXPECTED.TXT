 __  __    _    _   _ ____   ___
|  \/  |  / \  | \ | | __ ) / _ \
| |\/| | / _ \ |  \| |  _ \| | | |
| |  | |/ ___ \| |\  | |_) | |_| |
|_|  |_/_/   \_\_| \_|____/ \___/

____________________________________________________________
 Hello! I'm Manbo
 What can I do for you?
____________________________________________________________
____________________________________________________________
 Got it. I've added this task:
[T][ ] read
 Now you have 1 tasks in the list.
____________________________________________________________
____________________________________________________________
 Got it. I've added this task:
   [D][ ] submit (by: Friday)
 Now you have 2 tasks in the list.
____________________________________________________________
____________________________________________________________
 Got it. I've added this task:
   [E][ ] meeting (from: Monday 2pm to: 4pm)
 Now you have 3 tasks in the list.
____________________________________________________________
____________________________________________________________
 Here are the tasks in your list:
 1.[T][ ] read
 2.[D][ ] submit (by: Friday)
 3.[E][ ] meeting (from: Monday 2pm to: 4pm)
____________________________________________________________
____________________________________________________________
 Nice! I've marked this task as done:
[D][X] submit (by: Friday)
____________________________________________________________
____________________________________________________________
 OK, I've marked this task as not done yet:
[D][ ] submit (by: Friday)
____________________________________________________________
____________________________________________________________
 Nice! I've marked this task as done:
[T][X] read
____________________________________________________________
____________________________________________________________
 Here are the tasks in your list:
 1.[T][X] read
 2.[D][ ] submit (by: Friday)
 3.[E][ ] meeting (from: Monday 2pm to: 4pm)
____________________________________________________________
____________________________________________________________
 Bye. Hope to see you again soon!
____________________________________________________________
